File: pset1/mario.c
// Mario: receive integer height from user, then output a pyramid of that height.

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        printf("Height: ");
        n = get_int();
    }
    while (n < 0 || n > 23);
    
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        
        printf("  ");
      
        for (int l = 0; l < i + 1; l++)
        {
            printf("#");
        }
        
        printf("\n");
    }
    
    return 0;
}

File: pset6/mario.py
# Original program that prints a mario pyramid
# based on user inputted number. used as a te-
# -st writing session for new language Python.

import cs50

# prepare n
n = 0

# get an integer between 1 and 23 from user
while n < 1 or n > 23:
    
    print("Height: ", end="")
    n = cs50.get_int()
    
    if n > 0 and n < 24:
        break

# draw pyramid of size n
for i in range(0, n):
    
    for j in range(0, n - i - 1):
        print(" ", end="")
    
    for k in range(0, i + 1):
        print("#", end="")
    
    print("  ", end="")
    
    for l in range(0, i + 1):
        print("#", end="")
    
    print("")
